codelabs/custom_generator/custom_generator.md

```diff
@@ -68,7 +68,7 @@ The blocks are:
 Copy this code into `custom_generator.js` to define the two custom blocks.
 
 ```js
-Blockly.defineBlocksWithJsonArray([{
```

Comment:
Turns out this also need aliases in the file. See https://github.com/google/blockly/pull/5699